src/styles.py

```diff
@@ -40,27 +39,17 @@ def __init__(self, application):
         enumerator.close(None)
 
         config_dir = utilities.get_config_directory()
-        directory = config_dir.get_child_for_display_name("styles")
-        if not directory.query_exists(None):
-            directory.make_directory_with_parents(None)
-        system_stylenames = self._system_styles.keys()
-        enumerator = directory.enumerate_children("default::*", 0, None)
+        self._style_dir = config_dir.get_child_for_display_name("styles")
```

Comment:
Probably makes sense indeed, to make the style dir an attribute of the style manager class.